src/Core/BasePlugin.php

```diff
@@ -302,4 +310,15 @@ public function middleware(MiddlewareQueue $middlewareQueue): MiddlewareQueue
     public function services(ContainerInterface $container): void
     {
     }
+
+    /**
+     * Register application events.
+     *
+     * @param \Cake\Event\EventManagerInterface $eventsManager The global event manager to register listeners on
+     * @return \Cake\Event\EventManagerInterface
+     */
+    public function events(EventManagerInterface $eventsManager): EventManagerInterface
```

Comment:
```suggestion
    public function events(EventManagerInterface $eventManager): EventManagerInterface
```

Throughout the codebase we use the singular name `eventManager` for properties and methods.

Reply:
I have now replaced all the `eventsManager` occurrences with `eventManager`